doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask.rst

```diff
@@ -0,0 +1,140 @@
+.. lsst-task-topic:: lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask
+
+##################
+MakeDirectWarpTask
+##################
+
+Warp single visit images (calexps or PVIs) onto a common projection by
+performing the following operations:
+
+- Group the single-visit images by visit/run
+- For each visit, generate a Warp by calling method @ref run.
+
+.. _lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask-api:
+
+Python API summary
+==================
+
+.. lsst-task-api-summary:: lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask
+
+.. _lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask-subtasks:
+
+Retargetable subtasks
+=====================
+
+.. lsst-task-config-subtasks:: lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask
+
+.. _lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask-configs:
+
+Configuration fields
+====================
+
+.. lsst-task-config-fields:: lsst.pipe.tasks.make_direct_warp.MakeDirectWarpTask
+
+In Depth
+========
+
+Config Guidelines
+*****************
+
+The user must specify the size of the model PSF to
+which to match by setting `config.modelPsf.defaultFwhm` in units of pixels.
+The appropriate values depends on science case. In general, for a set of
+input images, this config should equal the FWHM of the visit with the worst
+seeing. The smallest it should be set to is the median FWHM. The defaults
+of the other config options offer a reasonable starting point.
+
+The following list presents the most common problems that arise from a
+misconfigured `~ip.diffim.modelPsfMatch.ModelPsfMatchTask`
+and corresponding solutions. All assume the default Alard-Lupton kernel,
+with configs accessed via
+`config.warpAndPsfMatch.psfMatch.kernel['AL']`. Each item in the list
+is formatted as:
+Problem: Explanation. *Solution*
+
+Troublshooting PSF-Matching Configuration
+*****************************************
+
+Matched PSFs look boxy
+**********************
```

Comment:
Seems like this should be different level of header, to nest below the "Troubleshooting ..." heading.

I also suspect it would be better to nest the `Solution` headers under their corresponding problem headers, rather than making them siblings.  That's especially true if those headings ever appear in any kind of table of contents, where just "Solution" with no context doesn't make any sense.